.chronus/changes/tspconfig-auto-complete-2024-9-18-13-48-34.md

```diff
@@ -0,0 +1,8 @@
+---
+changeKind: feature
+packages:
+  - "@typespec/compiler"
+  - typespec-vscode
+---
+
+Support completion for tspconfig.yaml file in vscode
```

Comment:
Those change do not match the PR description. If there is indeed still multiple issues being resolved(preferably those would be all separte prs are) every item must have it own changelog entry.

Reply:
PR description updated.